lambda/aws-qa-appsync-opensearch/question_answering/src/qa_agent/chain.py

```diff
@@ -95,21 +101,18 @@ def run_question_answering(arguments):
             return ''
 
         model_id=qa_model['modelId']
-        model_max_tokens = get_max_tokens(model_id)
+        model_provider=qa_model['provider']
+        model_max_tokens = get_max_tokens(model_provider+'.'+model_id)
```

Comment:
is this model_id coming from schema directly or is mapped with bedrockModel. from schema the user is expected to enter "anthropic.claude-3-sonnet-20240229-v1:0" and the get_max_token expect it to be BedrockModel.ANTHROPIC_CLAUDE_V3_SONNET_V1